roles/container_runtime/defaults/main.yml

```diff
@@ -21,7 +21,7 @@ openshift_docker_insecure_registries: []
 
 openshift_docker_ent_reg: 'registry.access.redhat.com'
 
-openshift_docker_options: False  # str
+openshift_docker_options: "--log-opt max-size=50m"  # str
```

Comment:
better to utilize openshift_docker_log_options for this, I think.

Reply:
That was our original thinking as well.  However, that ``openshift_docker_log_options`` is:

1. Not documented
2. Processed by a ton of black-magic-jinja-voodoo.
3. Complicated things, break easier than simple things.

As an admin of a cluster, infuriated by my limited log size, ``/etc/sysconfig/docker`` seems a good first-choice to look.  Esp. since I can see the ``max-size=50m`` in the ``ps`` output.  Also, ``max-size`` is specifically called out in the example, for the docs on ``openshift_docker_options``.

However, I always reserve my right to be dead-wrong :smile:   Why is ``openshift_docker_log_options`` better?

Reply:
openshift_docker_options and openshift_docker_log_options both end up in the same file.  Since we already have 'log_options' specifically, I think it's confusing to put log_options into the normal options area.

Reply:
@michaelgugino perhaps I'm misunderstanding "end up in the same file".  I believe ``openshift_docker_options`` affects ``/etc/sysconfig/docker`` and the other one does ``/etc/docker/docker.json``.  If you set the same option in both, the docker daemon will fail to start.  I think it's a stupid behavior to have, but it is documented, and we can only pick one :confounded:

Reply:
Clarification:  It's "documented" by docker.  We still have no documentation for ``openshift_docker_log_options`` that I could find.  So combined with it's added processing complexity, lead us to choose the simpler ``openshift_docker_options``.

Reply:
@cevich https://github.com/openshift/openshift-ansible/blob/master/roles/container_runtime/tasks/package_docker.yml#L109

Reply:
@michaelgugino Ohh it's right there :face_with_head_bandage:  How am I confusing this with ``docker.json``... @umohnani8 can u unscramble my brain?

Reply:
@umohnani8 oh nvm, remembered: it's in ``defailts/main.yml``...I see my confusion now.

@michaelgugino I think you intended to say:  ``openshift_docker_log_options`` and ``openshift_docker_options`` end up in BOTH ``/etc/docker/docker.json`` **AND** ``/etc/sysconfig/docker``, no?

Gosh that doesn't seem very POLA, and means ``openshift_docker_options`` is actually somewhat dangerous to use :confused:  Maybe we should add comment to this affect?  That would certainly help future engineers not waste time on it.

Reply:
@michaelgugino geeze...u right, I don't see any references to ``roles/container_runtime/templates/daemon.json``.  Shame on me for not checking that first.  Saw template...assumed use, bad me.  Should that file just be deleted then?  @umohnani8 I think just switch this option over to openshift_docker_log_options then (``False`` -> ``max-size=50m`` should be all that's needed IIRC).

Reply:
Sorry for being late to the discussion! Okay switching to `openshift_docker_log_options`.

Reply:
Fixed!